python/hsfs/feature_group.py

```diff
@@ -1283,9 +1283,9 @@ def as_of(self, wallclock_time, exclude_until=None):
             when calling the `insert()` method.
 
         # Arguments
-            wallclock_time: Datetime string. The String should be formatted in one of the
+            wallclock_time: timestamp, Datetime object or string. The String should be formatted in one of the
```

Comment:
One last comment about the docstrings. We should include all suppoerted types here, which are now:
`datatime.datetime, datetime.date, unix time in seconds (int), or string.` 
Also the docsstring shold be consistent across all parameters that have the `Optional[Union[str, int, datetime, date]]` input type. Right now, e.g. `FeatureView.train_validation_test_split` still has a different/old docstring.

Reply:
We also support timestamps is formats with spacing (e.g. `%Y-%m-%d`, `%Y-%m-%d %H`, `%Y-%m-%d %H:%M`, or `%Y-%m-%d %H:%M:%S`). But I don't think we can cover all the variants in the docstring of the function. Maybe we can document the possible variants in a guide and link there from the docstring (?)

Reply:
I created a Jira for this here https://hopsworks.atlassian.net/browse/FSTORE-316